components/builder-web/app/shared/project-settings/_project-settings.component.scss

```diff
@@ -33,6 +33,11 @@ hab-project-settings {
         &.cta {
           @include span-columns(3);
           text-align: center;
+
+          .mat-raised-button {
+            background-color: $hab-orange;
+            color: $hab-white;
+          }
```

Comment:
As you'll see below in the theme file, I changed the default accent from orange to blue which corrects all of the small form inputs/controls. In turn, I then had to override this call-to-action button to remain orange.